js/components/contact_download.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
```

Comment:
Just a code convention, but this component is really a [Page component](https://github.com/usdoj/foia.gov/blob/develop/docs/request-app.md#page) which should just be focused on gathering state and then propagating that state to a child component via props.